models/src_amazon_ads.yml

```diff
@@ -58,11 +58,11 @@ sources:
             description: "{{ doc('cost') }}"
 
       - name: advertised_product_report
-        description: Advertised product reports contain performance data for products that are advertised as part of your campaigns.
+        description: Advertised product reports contain performance data for products that are advertised as part of your campaigns. 
```

Comment:
What is the relationship between this table and product_ad_history?

Is it safe to assume the product_ad_history is just the history form of this report table?

Reply:
Yes, correct! I was wondering if in the staging models it should be remained to something like `product_ad_report` for consistency (assuming it is the correct table in the end.)